symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/MessageService.java

```diff
@@ -165,59 +170,67 @@ public TemplateEngine templates() {
    * @return a {@link V4Message} object containing the details of the sent message
    * @see <a href="https://developers.symphony.com/restapi/reference#create-message-v4">Create Message v4</a>
    */
+  @Deprecated
```

Comment:
Use Javadoc [`@deprecated`](https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html#javadoc_tag) tag to mention that this method will be replaced by `send(V4Stream, Message)`

Reply:
Should not we remove it at once since we are in beta?